specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -54,27 +54,27 @@ transaction capabilities to support such a scenario.
 
 This specification does not define how the
 components and the resources are partitioned or distributed within a
-single Java EE product. In order to achieve the transactional semantics
-required by the application, the Java EE Product Provider is free to
+single Jakarta EE product. In order to achieve the transactional semantics
+required by the application, the Jakarta EE Product Provider is free to
 execute the application components sharing a transaction in the same
 Java virtual machine, or distribute them across multiple virtual
 machines, in accordance with the requirements of the component
 specifications.
 
 The rest of this chapter describes the
-transactional requirements for a Java EE product in more detail. +
+transactional requirements for a Jakarta EE product in more detail. +
 
 === Requirements
 
 This section defines the transaction support
-requirements of Java EE Products that must be supported by Product
+requirements of Jakarta EE Products that must be supported by Product
 Providers.
 
 === Web Components
 
 Web components may demarcate transactions using
-the _javax.transaction.UserTransaction_ interface or transactional
-interceptors, which are defined in the JTA specification. They may
+the _jakarta.transaction.UserTransaction_ interface or transactional
```

Comment:
this shouldn't be changed in Jakarta EE 8 because it should be the same as Java EE 8.

Reply:
+1, good catch!

Reply:
This same issue appears a few times in the PR ("jakarta.t")